src/store/ducks/opensrp/locations/index.ts

```diff
@@ -48,7 +48,7 @@ export type LocationsActionTypes = FetchLocationsAction | RemoveLocationsAction
 
 /** interface to describe location state */
 export interface LocationsState {
-  locationsByPlanId: FlexObject<Location[]> | {};
+  locationsByPlanId: Dictionary<Location[]> | {};
```

Comment:
@kahummer what does this object look like?

Reply:
any key value pair object the FlexObject is pointing to '../../../../helpers/utils' instead of using the Dictionary equivalent added by js-tools ona@utils

Reply:
I mean this `Dictionary<Location[]>` - what data fits this type def?

Reply:
I think it's array with location data keyed by id
maybe 
```
{
'ed2b4b7c-3388-53d9-b9f6-6a19d1ffde1f': [
identifier: string,
name: string
]
}
```

Reply:
cool